scripts/acn/k8s/dns.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: externaldns.k8s.io/v1alpha1
+kind: DNSEndpoint
+metadata:
+  name: deployment-name-here 
+  namespace: agents-p2p-dht
+spec:
+  endpoints:
+  - dnsName:  agents-p2p-dht.sandbox.fetch-ai.com
```

Comment:
placeholder instead?

Reply:
yes, all your placeholders suggestions are correct.
I just added the files from the other repo where only node related properties are configurable.
Didn't do the same for platform properties.